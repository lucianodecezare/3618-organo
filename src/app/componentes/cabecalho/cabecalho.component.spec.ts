import { ComponentFixture, TestBed } from '@angular/core/testing';
import { CabecalhoComponent } from './cabecalho.component';

describe('CabecalhoComponent', () => {
  let component: CabecalhoComponent;
  let fixture: ComponentFixture<CabecalhoComponent>;

  beforeEach(() => {
    TestBed.configureTestingModule({
      imports: [CabecalhoComponent],
    });

    fixture = TestBed.createComponent(CabecalhoComponent);
    component = fixture.componentInstance;
  });

  it('should be created', () => {
    expect(component).toBeTruthy();
  });

  it('should define alt and src properties', () => {
    expect(component.alt).toBeDefined();
    expect(component.src).toBeDefined();
  });

  it('should render content based on src and alt', () => {
    component.src = 'https://example.com/test-image.jpg';
    component.alt = 'Imagem teste';

    expect(component).toMatchSnapshot();
  });
});
