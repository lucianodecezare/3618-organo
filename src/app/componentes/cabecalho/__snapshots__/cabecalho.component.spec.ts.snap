// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CabecalhoComponent should render content based on src and alt 1`] = `
CabecalhoComponent {
  "__ngContext__": 4,
  "alt": "Imagem teste",
  "src": "https://example.com/test-image.jpg",
}
`;
